src/docs/development/tools/devtools/memory.md

```diff
@@ -38,77 +47,100 @@ in granularity:
   requiring more information.
 {{site.alert.end}}
 
-## Memory anatomy
+## Memory Anatomy
 
-Use a timeseries graph to visualize the state of the Flutter memory
+A timeseries graph is used to visualize the state of the Flutter memory
 at successive intervals of time. Each data point on the chart
 corresponds to the timestamp (x-axis) of measured quantities (y-axis)
 of the heap, for example, usage, capacity, external, garbage
 collection, and resident set size.
 
-![Screenshot of a memory anatomy page]({% asset tools/devtools/memory_anatomy.png @path %}){:width="100%"}
+![Screenshot of a memory anatomy page]({% asset tools/devtools/memory_chart_anatomy.png @path %}){:width="100%"}
+
+The major sections of the memory profiler are:
 
 <dl markdown="1">
-<dt markdown="1">**Legend**</dt>
-<dd>All collected measurements regarding the memory.
-    Clicking a legend name hides or shows that data.</dd>
-<dt markdown="1">**Range selector**</dt>
+<dt markdown="1">**Event Timeline**</dt>
+<dd>Displays DevTools events (such as Snapshot (manual and auto), Dart VM GC, user requested GC, Monitor and Accumulator Reset (button clicks) when they occurred, in relation to the memory chart timeline.</dd>
+<dt markdown="1">**Memory Overview Chart**</dt>
+<dd>This chart is a timeseries graph to help visualize the state of the Dart heap and External Heaps at various points in time.</dd>
```

Comment:
r/to help visualize/that visualizes
r/External Heaps/external heaps
r/at various points in time/over time

Reply:
Done.